src/models/tests/pairwise_union_reach_surface_test.py

```diff
@@ -0,0 +1,114 @@
+# Copyright 2021 The Private Cardinality Estimation Framework Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for pairwise_union_reach_surface.py."""
+
+from absl.testing import absltest
+import numpy as np
+from wfa_planning_evaluation_framework.models.reach_point import ReachPoint
+from wfa_planning_evaluation_framework.models.reach_surface import ReachSurface
+from wfa_planning_evaluation_framework.models.pairwise_union_reach_surface import PairwiseUnionReachSurface
+from wfa_planning_evaluation_framework.models.reach_curve import ReachCurve
+
+
+class LinearCappedReachCurve(ReachCurve):
```

Comment:
You could probably just use GoergModel rather than inventing a new class.

Reply:
I took it from the reach_curve_test. If you don't mind I would like to keep this since it is vert intuitive?
But, I can change if you think there is value in using GoergModel model here.